server/pulp/server/db/model/dispatch.py

```diff
@@ -489,3 +489,26 @@ def __init__(self, call_request, call_report):
         self.timestamp = dateutils.now_utc_timestamp()
         self.call_request_string = str(call_request)
         self.serialized_call_report = call_report.serialize()
+
+
+class CeleryBeatLock(Model):
```

Comment:
Convert this to inherit from `Document`. Document should come from `mongoengine`. Grep the codebase for existing objects that inherit from `Document` as examples.